.github/workflows/pr-push.yml

```diff
@@ -0,0 +1,22 @@
+name: Node CI
+
+on:
+  push:
+    branches:
+      - gh-pages
+  pull_request: {}
+
+jobs:
+  validate-and-publish:
+    name: Validate and Publish
+    runs-on: ubuntu-latest # only linux supported at present
+    steps:
+      - uses: actions/checkout@v2
+      - uses: sidvishnoi/spec-prod@v2
+        with:
+          TOOLCHAIN: respec
+          W3C_ECHIDNA_TOKEN: ${{ secrets.ECHIDNA_TOKEN }}
+          W3C_WG_DECISION_URL: "https://lists.w3.org/Archives/Public/public-device-apis/2021May/0008.html"
+          W3C_NOTIFICATIONS_CC: "${{ secrets.CC }}"
+          W3C_BUILD_OVERRIDE: |
+            specStatus: WD
```

Comment:
@marcoscaceres Should this really be WD, btw? (from https://github.com/w3c/browser-specs/issues/439)

Reply:
Yeah... because ... reasons... it will be a living REC again soon, and we can fix the whole short name mess with this spec.